package.json

```diff
@@ -70,6 +70,11 @@
 					"type": "boolean",
 					"default": true,
 					"description": "Forces indenting with two spaces when Dart files are opened. This is on by default because VS Code doesn't support per-language settings and most people use tabs/4 spaces for other languages."
+				},
+				"dart.showTodos": {
+					"type": "boolean",
+					"default": false,
```

Comment:
I guess I'm leaning towards enabling it by default, and assuming that people will see the setting if they want to disable it.

Right now it takes a restart before the setting really takes effect - it filters as the analysis errors are created, and doesn't remember them to re-report to vscode.